webapp/channels/src/components/team_sidebar/team_sidebar.tsx

```diff
@@ -205,7 +205,7 @@ export default class TeamSidebar extends React.PureComponent<Props, State> {
         const sortedTeams = filterAndSortTeamsByDisplayName(this.props.myTeams, this.props.locale, this.props.userTeamsOrderPreference);
 
         const currentProduct = getCurrentProduct(this.props.products, this.props.location.pathname);
-        if (currentProduct && !currentProduct.showTeamSidebar) {
+        if (!this.props.alwaysShowTeamSidebar && currentProduct && !currentProduct.showTeamSidebar) {
```

Comment:
I'm not sure about this part. This feature was mainly requested for the "Channels" product. I'm not sure what the implications are to show the sidebar, when a product would otherwise not be showing it.

Reply:
I'm thinking we should remove this change, as the point of this feature is to show when the user is a member of one team. I think overriding a product's decision to show the sidebar is outside of the scope of this feature.

Reply:
I would agree this change should be removed. While both playbooks and boards currently show the team sidebar, overriding that here probably isn't correct.